bus_presence_override/static/src/js/systray.js

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2017 LasLabs Inc.
+   License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+odoo.define('bus_presence_override.systray', function (require) {
+    "use strict";
+
+    var DataModel = require('web.DataModel');
+    var session = require('web.session');
+    var SystrayMenu = require('web.SystrayMenu');
+    var Widget = require('web.Widget');
+
+    var Systray = Widget.extend({
+        template:'systray',
+        events: {
+            'click .o_user_presence_status': 'on_click_user_presence_status',
+        },
+        init: function() {
+            this._super.apply(this, arguments);
+            this.Partners = new DataModel('res.partner');
+            this.status_icons = {
+                'online': 'fa fa-circle o_user_online',
+                'away': 'fa fa-circle o_user_idle',
+                'offline': 'fa fa-circle-o',
+            }
+        },
+        start: function () {
+            this._update_im_status_custom(status='online');
+            return this._super()
+        },
+        on_click_user_presence_status: function (event) {
```

Comment:
use lowerSnakeCase for anything you would typically snake_case in Python. Variable names, methods, etc.